### PR TITLE
Add cargo-audit config

### PR DESCRIPTION
**Pull request by ionut-arm**
Signed-off-by: Ionut Mihalcea <ionut.mihalcea@arm.com>

### CODE REVIEW

File: .cargo/audit.toml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+[target]
+arch = "x86_64" # Ignore advisories for CPU architectures other than this one
+os = "linux" # Ignore advisories for operating systems other than this one
+
+[packages]
+source = "all" # "all", "public" or "local"
+
+[yanked]
+enabled = true # Warn for yanked crates in Cargo.lock
+update_index = true # Auto-update the crates.io index
```
Comment (hug-dev) [inline .cargo/audit.toml:31]: Is all of this similar than the default configuration?

Comment (ionut-arm) [reply .cargo/audit.toml:31]: Yes, sorry, should've linked - this is just copied from their default with a couple of changes: https://github.com/RustSec/rustsec/blob/main/cargo-audit/audit.toml.example

Comment (ionut-arm) [reply .cargo/audit.toml]: Done, I've added `unmaintaned` to the denied list.

**Review by hug-dev — APPROVED** [review]
👌

**Pull request merged**